db/migrate/20191118120312_add_destroy_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDestroyTokenToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :destroy_token, :string
```

Comment:
Given our giant user table and the fact that this token is very temporary, what do you think about storing this somewhere like Redis instead? Then when a user clicks the link we could check Redis for the token and if present for that user and they match we delete the user and the token. Having it in Redis means we could also expire it and set a time limit and tell the user you have 6 hours to delete your account. This I think adds another nice little layer of security.